### DISCUSSION TITLE
callback - next-auth / nextjs

### DISCUSSION THREAD

**Discussion by Batman4234** [vercel/community#380] [Help] [answered]
i get the error "https://myapp/continue?callbackUrl=myapp/&error=Callback", happens only with vercel. i use prisma as my db with postgresql

**Comment by swarnava** [comment, marked as answer]
Hi there,

Request you to create an issue on the Next auth GitHub repo with reproduction steps and additional details so the team can triage this: https://github.com/nextauthjs/next-auth/issues

Have a good one!
